x-pack/test/cloud_security_posture_functional/config.ts

```diff
@@ -13,36 +13,36 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
   const xpackFunctionalConfig = await readConfigFile(
     require.resolve('../functional/config.base.js')
   );
-
-  return {
-    ...xpackFunctionalConfig.getAll(),
-    pageObjects,
-    testFiles: [resolve(__dirname, './pages')],
-    junit: {
-      reportName: 'X-Pack Cloud Security Posture Functional Tests',
-    },
-    kbnTestServer: {
-      ...xpackFunctionalConfig.get('kbnTestServer'),
-      serverArgs: [
-        ...xpackFunctionalConfig.get('kbnTestServer.serverArgs'),
-        /**
-         * Package version is fixed (not latest) so FTR won't suddenly break when package is changed.
-         *
-         * test a new package:
-         * 1. build the package and start the registry with elastic-package and uncomment the 'registryUrl' flag below
-         * 2. locally checkout the kibana version that matches the new package
-         * 3. update the package version below to use the new package version
-         * 4. run tests with NODE_EXTRA_CA_CERTS pointing to the elastic-package certificate
-         * 5. when test pass:
-         *   1. release a new package to EPR
-         *   2. merge the updated version number change to kibana
-         */
-        `--xpack.fleet.packages.0.name=cloud_security_posture`,
-        `--xpack.fleet.packages.0.version=1.7.4`,
-        // `--xpack.fleet.registryUrl=https://localhost:8080`,
-        `--xpack.fleet.agents.fleet_server.hosts=["https://ftr.kibana:8220"]`,
-        `--xpack.fleet.internal.fleetServerStandalone=true`,
-      ],
-    },
-  };
+  // If TEST_CLOUD environment varible is defined, return the configuration for cloud testing
+  if (process.env.TEST_CLOUD !== '1') {
```

Comment:
instead of if statement please refactor the following into a new config file: `config.cloud.ts`

Reply:
fixed